app/src/src/components/common/FormatDate/toYearMonthDayInNumbers.jsx

```diff
@@ -6,6 +6,14 @@ const toYearMonthDayInNumbers = ({ selectedValue }) => {
     month: 'numeric',
     day: 'numeric',
   });
-  return YYYYMMDD;
+  const check = YYYYMMDD.indexOf('/');
+  // on android, it returns with / instead of - in format of MM/DD/YY instead of YYYY-MM-DD
+  if (check == -1) return YYYYMMDD;
+  else {
+    const dateArr = YYYYMMDD.split('/');
+    const androidYYYYMMDD =
+      '20' + dateArr[2] + '-' + dateArr[0] + '-' + dateArr[1];
+    return androidYYYYMMDD;
+  }
```

Comment:
You can take everything inside this else statement and move it outside of the else block. It's safe to do so because the if statement is returning a value, so it's implied that anything after the return is part of the else.